backend/src/services/muralcollection.service.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  MuralCollection,
+  MuralCollectionInterface,
+} from "../models/muralcollection.model";
+import { Mural } from "../models/mural.model";
+import { UpdateOptions } from "sequelize";
+
+export class MuralCollectionService {
+  public async create(collecton: MuralCollectionInterface, murals: number[]) {
+    const createdCollection: MuralCollection = await MuralCollection.create<MuralCollection>(
+      collecton
+    );
+    murals.forEach(async (muralId) => {
+      const mural = await Mural.findByPk<Mural>(muralId, {
+        rejectOnEmpty: true,
+      });
+      createdCollection.addMural(mural);
```

Comment:
Nice use of the Postgres association methods!

Reply:
Thanks!